src/api/api.js

```diff
@@ -37,7 +35,7 @@ async function createEvent(event) {
 
 async function getEvent(id) {
   try {
-    const { data } = await protectedResourceAxios.get(`${baseUrl}/protected/events/${id}`);
+    const { data } = await protectedResourceAxios.get(`${process.env.VUE_APP_API_DOMAIN}/api/v1/protected/events/${id}`);
```

Comment:
I'd prefer to define this as baseUrl like above.

Reply:
I agree, but we should build a more complete constant structure to do it with maps and stuff